tests/column2vec/test_column2vec.py

```diff
@@ -0,0 +1,296 @@
+import os
+import unittest
+import time
+
+import pandas as pd
+from sentence_transformers import SentenceTransformer
+
+from column2vec.src.column2vec import (column2vec_as_sentence, column2vec_as_sentence_clean,
+                            column2vec_as_sentence_clean_uniq, column2vec_avg,
+                            column2vec_weighted_avg, column2vec_sum,
+                            column2vec_weighted_sum, cache)
+from column2vec.src.functions import get_nonnumerical_data, get_clusters, compute_distances
+from similarity_framework.src.impl.metadata.type_metadata_creator import TypeMetadataCreator
+from similarity_framework.src.models.types_ import NONNUMERICAL
+
+SKIP_CLUSTERS = True
+SKIP_SIMILAR = False
+# MODEL = 'all-mpnet-base-v2'  # bert-base-nli-mean-tokens
+MODEL = 'bert-base-nli-mean-tokens'  #
```

Comment:
is this decision final, maybe remove commented part, or provide as alternative in comment if intended to be used in future

Reply:
fixed